numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -3840,9 +3840,6 @@ def add_newdoc(place, name, doc):
 
```

Comment:
Might make more sense to remove this one

Reply:
I agree, it makes more sense to me to remove the first instance of `x//4` and to leave the last one for comparison with the true division operator.

Reply:
Agree also. @emmanuelle Ping.